docs/system/k8s-metrics.md

```diff
@@ -0,0 +1,168 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Container
+--->
+
+# Semantic Conventions for Kubernetes Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+## K8s Metrics
+
+This document describes instruments and attributes for common K8s level
```

Comment:
could we document which resources we expect to be used with these metrics?

Reply:
Could you clarify what "resources" stands for here? Pods and Nodes are K8s resources, do you refer to this or sth else :) ?

Reply:
I mean that these metrics does not make any sense without specific resources (nods and pods). There is nothing in this document that talks about it.

I'd suggest to say something similar to 

https://github.com/open-telemetry/semantic-conventions/blob/46b1dffba6187e0f4cc8d59a48f8544fb84f11f3/docs/system/hardware-metrics.md?plain=1#L49-L50

Reply:
That makes sense, I believe most (hopefully all) of the upcoming metrics will be related to some of the K8s resources.